discussion/app/controllers/CommentsController.scala

```diff
@@ -155,7 +155,6 @@ object CommentsController extends DiscussionController with ExecutionContexts {
     }
       .recover {
         case NonFatal(e) =>
-          log.error(s"Error fetching comments from request with following headers: ${request.headers}", e)
```

Comment:
Also removing this logging which was added to find out where traffic which is crawling our discussions was coming from.